test/ClientTest.cpp

```diff
@@ -557,6 +579,80 @@ single_conn_call(Connector<BUFFER, NetProvider> &client)
 	client.close(conn);
 }
 
+/** Single connection, separate executes */
+template <class BUFFER, class NetProvider = Net_t>
+void
+single_conn_sql_statements(Connector<BUFFER, NetProvider> &client)
+{
+	TEST_INIT(0);
+	Connection<Buf_t, NetProvider> conn(client);
+	int rc = client.connect(conn, localhost, port);
+	fail_unless(rc == 0);
+
+	TEST_CASE("CREATE TABLE");
+	rid_t create_table = conn.execute("CREATE TABLE IF NOT EXISTS testing_sql "
+									  "(column1 UNSIGNED PRIMARY KEY, "
+									  "column2 VARCHAR(50), "
```

Comment:
Add also test cases including collations, autoincrements, different spans etc so that cover all possible values of metadata.